docker/tools/Dockerfile

```diff
@@ -45,6 +45,10 @@ COPY --from=builder /diem/target/release/db-restore /usr/local/bin
 COPY --from=builder /diem/target/release/diem-transaction-replay /usr/local/bin
 COPY --from=builder /diem/target/release/diem-writeset-generator /usr/local/bin
 
+### Get Move modules bytecodes for genesis ceremony
+RUN mkdir -p diem/move
+COPY /diem/language/diem-framework/releases/artifacts/current diem/move
```

Comment:
You need to COPY from the previous build step: e.g. `COPY --from=builder /diem/language/diem-framework/releases/artifacts/current /diem/move`

also nit: use `/diem/move` instead of `diem/move`